src/app/src/main/resources/config.properties

```diff
@@ -5,7 +5,7 @@ adr_pop_schema=inputs-unaids-population
 adr_shape_schema=inputs-unaids-geographic
 adr_survey_schema=inputs-unaids-survey
 adr_schema=inputs-unaids-estimates
-adr_url=https://dev.adr.fjelltopp.org/
```

Comment:
This should remain as `dev.adr.fjelltop.org` as it's the test ADR site we use for local testing. The real url gets injected into the config during deployment. All automated tests are run against the dev server where there is a test user and fake data, so changing this will cause them all to fail.